pdfplumber/table.py

```diff
@@ -450,6 +450,11 @@ def __init__(self, page, settings={}):
         )
         self.cells = intersections_to_cells(self.intersections)
         self.tables = [Table(self.page, t) for t in cells_to_tables(self.cells)]
+        # If multiple tables are found, sort them by their vertical placement.
+        # Top to bottom, left to right.
+        self.tables = sorted(
```

Comment:
I'd rather not assign `self.tables` twice. And, given that we're just sorting a list returned by `cells_to_tables(...)`, I think the sorting probably fits best in that function. 

In fact, there's already a line there that handles sorting: https://github.com/jsvine/pdfplumber/blob/de767c3a52bbcacd237a17a051a3975d592023d9/pdfplumber/table.py#L315

That logic identifies each table's leftmost-and-then-topmost cell (due to the `(left, top, right, bottom)` sequence of the bbox tuples), and then does the same comparison between tables. I think you're right, though, that topmost-and-then-leftmost is a more reasonable expectation, especially when the left-alignment of a series of tables might differ by just a few (otherwise imperceptible) decimal places.

Reply:
Yeah, I wasn't sure what the `min(t["corners"])` was doing. Thanks for explaining. Here's what I have done: Added `[::-1]` to it so that the tuple gets reversed to become `(y, x)` instead of `(x, y)`. Removed the additional `sorted(...)` call too.